docs/reference/cluster/nodes-info.asciidoc

```diff
@@ -54,6 +54,9 @@ the operating system:
 `os.available_processors`::
 	Number of processors available to the Java virtual machine
 
+`os.allocated_processors`::
+    Number of processors that ES is using. Imply the `processors` value passed in settings, at most 32
```

Comment:
Can we reword this to `The number of processors actually used to calculate thread pool size. This number can be set with the `processors` setting of a node and defaults to the number of processors reported by the OS. In both cases this number will never be larger than 32.